docs/UserGuide.md

```diff
@@ -72,6 +72,14 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Onboarding guide
+The user will be prompted to take a quick tour of the ContacX's basic functions via a series of step-by-step instructions on the first run of the program.
```

Comment:
Should a friendlier tone be used? E.g.
> When you run the program for the first time, you will be prompted to ...

Not a must, but I feel it better fits the rest of the user guide

Reply:
Good suggestion! Will update accordingly.